tyndale-bulletin.csl

```diff
@@ -284,6 +295,17 @@
       <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
         <text variable="title" font-style="italic" text-case="title"/>
       </else-if>
+      <!-- Include the reviewed author for book reviews per §§11.3.8&#8211;11.3.9. -->
+      <else-if variable="reviewed-author">
+        <choose>
+          <if variable="reviewed-title">
+            <text variable="title" quotes="true" suffix=", " text-case="title"/>
+            <text variable="reviewed-title" font-style="italic" prefix="review of " text-case="title"/>
+          </if>
+        </choose>
+        <text variable="title" font-style="italic" prefix="Review of " suffix=", by " text-case="title"/>
```

Comment:
This should be an `else` condition, no? Otherwise you get the title twice when there is a reviewed title

Reply:
It does. Thanks. Sorry I missed that. I'll resubmit presently.